include/psa/crypto_extra.h

```diff
@@ -2069,6 +2069,264 @@ static inline struct psa_pake_operation_s psa_pake_operation_init(void)
     return v;
 }
 
+typedef enum {
+/** An invalid key data format identifier. */
+    PSA_KEY_DATA_FORMAT_NONE,
```

Comment:
Is it particularly helpful to include 'DATA' in the type and value identifiers? - I think that `PSA_KEY_FORMAT_XXX` and `psa_key_format_t` would be unambiguous enough.